lib/rust/ensogl/component/grid-view/src/column_widths.rs

```diff
@@ -0,0 +1,91 @@
+//! Storage for column's widths in the GridView. See [`ColumnWidths`].
+
+use crate::prelude::*;
+use std::fmt::Formatter;
+
+
+
+// ====================
+// === ColumnWidths ===
+// ====================
+
+
+// === Helper definitions ===
```

Comment:
```suggestion
// ====================

// === Helper definitions ===
```
We don't add additional space between headers.

Reply:
Fixed, though I wish `enso-formatter` would handle that. (it does not at the moment)